source/iotagent-architecture.rst

```diff
@@ -515,14 +515,10 @@ and vice-versa.
 Libraries to assist the development of new IotAgents
 ====================================================
 
-We have libraries that abstract some points describe in previous topics
+We have libraries that abstract some points described in previous topics
 to facilitate the development of an IotAgent.
 
 There are two libraries:
 
  - node.js **recommended** (https://www.npmjs.com/package/@dojot/iotagent-nodejs)
```

Comment:
@dojot/developers   Would be nice replace this reference to "dojot-microservice-sdk-js" ?

Reply:
I think not, because sdk is not a iotagent lib..